docs/docs/data/create.tsx

```diff
@@ -534,11 +534,23 @@ const createEndpointTableData = [
         A list of the BigBlueButton client plugins you want included for the
         specific session (merged with the list in <code>/etc/bigbluebutton/bbb-web.properties</code> and duplicates dropped){" "}
         <code className="language-plaintext highlighter-rouge">
-          {`pluginManifests=[{"url":"https://someserver.com/plugins/bbb-plugin-pick-random-user/manifest.json"}]`}
+          {`pluginManifests=[{"url":"https://someserver.com/plugins/bbb-plugin-pick-random-user/manifest.json", "checksum": "abc123"}]`}
```

Comment:
Is this `checksum` referenced anywhere else? First time I see it...

Reply:
Hmm, good point, I must have forgotten to mention it. I will fix it in the docs soon enough